elftools/elf/descriptions.py

```diff
@@ -144,6 +164,34 @@ def describe_note(x):
                       _DESCR_NOTE_N_TYPE.get(x['n_type'], _unknown))
     return '%s (%s)%s' % (note_type, note_type_desc, desc)
 
+
+def describe_attr_tag_arm(tag, val, extra):
+    idx = ENUM_ATTR_TAG_ARM[tag] - 1
+    d_entry = _DESCR_ATTR_VAL_ARM[idx]
+
+    if d_entry is None:
+        if tag == 'TAG_COMPATIBILITY':
+            return (_DESCR_ATTR_TAG_ARM[tag]
+                    + 'flag = %d, vendor = %s' % (val, extra))
+
+        elif tag == 'TAG_ALSO_COMPATIBLE_WITH':
+            if val.tag == 'TAG_CPU_ARCH':
+                return _DESCR_ATTR_TAG_ARM[tag] + d_entry[val]
```

Comment:
I noticed the following bug while working on #514: `d_enty is None` in line 172, so this will fails:
> Value of type "None" is not indexable  [index]